apps/prairielearn/src/pages/publicAssessmentQuestionsPreview/publicAssessmentQuestionsPreview.sql

```diff
@@ -0,0 +1,90 @@
+-- BLOCK select_assessment_by_id
+SELECT
+  *
+FROM
+  assessments
+where
+  id = $assessment_id;
+
+-- BLOCK check_assessment_is_public
+SELECT
+  a.shared_publicly
+FROM
+  assessments AS a
+WHERE
+  a.id = $assessment_id;
+
+-- BLOCK check_course_instance_is_public
+SELECT
+  ci.shared_publicly
+FROM
+  course_instances AS ci
+WHERE
+  ci.id = $course_instance_id;
+
+
+-- BLOCK questions
+SELECT
+  aq.*,
+  q.qid,
+  q.title,
```

Comment:
This is identical to the query in `apps/prairielearn/src/pages/instructorAssessmentQuestions/instructorAssessmentQuestions.sql` except for not having the issue count. Should we instead factor this query out to `models/questions.ts` and call it `selectQuestionsForAssessment` or something like that?